iina/PlayerCore.swift

```diff
@@ -668,11 +668,18 @@ class PlayerCore: NSObject {
   }
 
   func frameStep(backwards: Bool) {
+    // When playback is paused the display link is stopped in order to avoid wasting energy on
+    // It must be running when stepping to avoid slowdowns caused by mpv waiting for IINA to call
+    // mpv_render_report_swap.
+    mainWindow.videoView.displayActive()
```

Comment:
It seems even if the display link is not running, frame step commands can run correctly. Do we have to activate the display link manually?